odoo/orm/fields_selection.py

```diff
@@ -227,9 +227,6 @@ def convert_to_cache(self, value, record, validate=True):
         raise ValueError("Wrong value for %s: %r" % (self, value))
 
     def convert_to_export(self, value, record):
-        if not isinstance(self.selection, list):
-            # FIXME: this reproduces an existing buggy behavior!
-            return value if value else ''
```

Comment:
Some selection can be callable and sometime this callable doesn't contains the value/label pair in the result (because it may depends of external data or time).
In that case, IMO, it is preferable to return the raw value than empty string (fallback of the return) :thinking: what do you think ?

Reply:
Sounds good 👍🏻 
I've pushed the change